examples/webgl_materials_envmaps_hdr.html

```diff
@@ -215,12 +219,53 @@
 
 				window.addEventListener( 'resize', onWindowResize, false );
 
+				function updateEnvMap() {
+
+					var newEnvMap = standardMaterial.envMap;
+
+					switch( params.envMap ) {
+
+						case 'None': newEnvMap = null; break;
+						case 'LDR': newEnvMap = ldrCubeRenderTarget.texture; break;
+						case 'HDR': newEnvMap = hdrCubeRenderTarget.texture; break;
+						case 'RGBM16': newEnvMap = rgbmCubeRenderTarget.texture; break;
+
+					}
```

Comment:
The `updateEnvMap` refactor is slightly tidying, but, hey, it's just an example. I'd just take the commit out and replace `undefined` with `null`. Then let's merge it :).